packages/next/src/build/webpack/plugins/middleware-plugin.ts

```diff
@@ -127,6 +128,10 @@ function getEntryFiles(
     files.push(`server/edge-${INSTRUMENTATION_HOOK_FILENAME}.js`)
   }
 
+  if (process.env.NODE_ENV === 'production') {
+    files.push(PRERENDER_MANIFEST.replace('json', 'js'))
+  }
```

Comment:
Are you sure this isn't needed for dev?

Is that because we keep the generated id in memory during dev?

Reply:
@styfle we don't currently write this manifest to disk in dev so it falls back to using `development-id` as the preview id. Will this cause problems?